src/leap/mail/imap/tests/test_imap.py

```diff
@@ -96,9 +88,10 @@ def setUp(self):
         MessageCollection interface in this particular TestCase
         """
         super(MessageCollectionTestCase, self).setUp()
```

Comment:
true. I haven't payed attention to this testcase, it probably have to be redone or moved to some other module. added TODO